rust/candid/src/de.rs

```diff
@@ -17,6 +17,9 @@ use serde::de::{self, Visitor};
 use std::fmt::Write;
 use std::{collections::VecDeque, io::Cursor, mem::replace};
 
+pub mod arc;
+pub mod rc;
```

Comment:
Is it better to put the module in `lib.rs` instead of `de.rs`, as they contain both serialize and deserialize?

Reply:
Sure, done!